### PR TITLE
Bug/autocomplete 2/#58

### PR DESCRIPTION
**Pull request by abaliunov-sc**

### CODE REVIEW

File: src/client/components/SlateEditor/plugins/slate-autocomplete-plugin/AutocompleteContainer.js [hunk 1]
```diff
@@ -112,6 +121,10 @@ class AutocompleteContainer extends React.Component {
     }
   };
 
+  // handleScroll = () => {
```
Comment (kvolkovich-sc) [inline src/client/components/SlateEditor/plugins/slate-autocomplete-plugin/AutocompleteContainer.js]: Remove comments

File: src/client/components/SlateEditor/plugins/slate-autocomplete-plugin/AutocompleteContainer.js [hunk 2]
```diff
@@ -171,6 +193,9 @@ class AutocompleteContainer extends React.Component {
             items={items}
             isLoading={isLoading}
             selectedIndex={selectedIndex}
+            onMouseDown={this.props.onMouseDown}
+            onScroll={this.props.onScroll}
+            // onScroll={this.handleScroll.bind(this)}
```
Comment (kvolkovich-sc) [inline src/client/components/SlateEditor/plugins/slate-autocomplete-plugin/AutocompleteContainer.js]: Remove comment

File: src/client/components/PlainMarkdownInput/PlainMarkdownInput.react.js [hunk 3]
```diff
@@ -166,6 +184,19 @@ class PlainMarkdownInput extends React.Component {
     return change;
   }
 
+  handleRef(ref) {
+    this.slateContentRef = ref;
+  }
+
+  handleScroll() {
+    if (navigator.userAgent.match(/msie/i) || navigator.userAgent.match(/trident/i)) {
```
Comment (kvolkovich-sc) [inline src/client/components/PlainMarkdownInput/PlainMarkdownInput.react.js]: Add comment with explanations

File: src/client/components/SlateEditor/plugins/slate-autocomplete-plugin/AutocompleteContainer.js (hunk 1 above)
Comment (abaliunov-sc) [reply src/client/components/SlateEditor/plugins/slate-autocomplete-plugin/AutocompleteContainer.js]: Done

File: src/client/components/SlateEditor/plugins/slate-autocomplete-plugin/AutocompleteContainer.js (hunk 2 above)
Comment (abaliunov-sc) [reply src/client/components/SlateEditor/plugins/slate-autocomplete-plugin/AutocompleteContainer.js]: Done

File: src/client/components/PlainMarkdownInput/PlainMarkdownInput.react.js (hunk 3 above)
Comment (abaliunov-sc) [reply src/client/components/PlainMarkdownInput/PlainMarkdownInput.react.js]: Done

**Comment by abaliunov-sc** [comment]
It is corrected

**Pull request merged**